addons/survey/models/survey_question.py

```diff
@@ -275,6 +280,49 @@ def _compute_validation_required(self):
             if not question.validation_required or question.question_type not in ['char_box', 'numerical_box', 'date', 'datetime']:
                 question.validation_required = False
 
+    @api.depends('is_conditional', 'survey_id', 'survey_id.question_ids', 'triggering_question_id')
+    def _compute_allowed_triggering_question_ids(self):
+        """ This method is required to fetch the possible triggering questions when
+        the question is being created. """
```

Comment:
Could mention here why we don't depend on the sequence field (to avoid RPCs cascades when re-ordering questions on the frontend etc)